lib/openrouter.py

```
@@ -106,6 +107,118 @@ def synthesize_idea(
+                                {"role": "system", "content": SYSTEM_PROMPT},
+                                {"role": "user", "content": user_prompt},
+                            ],
+                            "temperature": 0.7,
+                            "max_tokens": 2000,
+                            "response_format": {"type": "json_object"},
+                        },
+                        timeout=30,
+                    )
+                    response.raise_for_status()
+                    break
+                except Exception as fe:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)